demo/demo.md

```diff
@@ -244,7 +254,8 @@ The following example illustrates a change of gauge flight with a layover in SEA
 
 ## Using the footer slot
 
-In this example for a flight that requires government approval or a flight that is operated by another subsidiary or partner carrier, you can use the `footer` custom element slot to insert additional information into the scope of the component. In the code you will see the use of `<auro-icon>` and text within the named slot element.
+In this example for a flight that requires government approval or a flight that is operated by another subsidiary or partner carrier, you can use the `footer` custom element slot to insert additional information into the scope of the component. In the code you will see the use of `<auro-icon>` and text within the named slot element. This slot requires the consumer to control
+what they intend to be read by a screen reader manually through the use of `aria-hidden="true"`.
```

Comment:
Consumers have to be in control of what they want read to a guest with a screen reader when they slot something into the foote. If you don't put aria-hidden on some elements, it can cause the VoiceOver to start reading things out like "operational disclosure text" (input voiceover navigate command) > reads the same line again + "group".